### PR TITLE
Fix input styles

### PR DESCRIPTION
**Pull request by gordonbrander**
Fixes https://github.com/mozilla/browser.html/issues/826.

- Move icon padding from input wrapper to input. This won't fix the dotted focus line in Servo, but it will at least make it wrap around the input instead of a random box.
- Add focus border to input.

### CODE REVIEW

File: src/browser/input.js [hunk 1]
```diff
@@ -219,9 +220,7 @@ export const view/*:type.view*/ = (model, address) =>
   html.div({
     className: 'input-combobox',
     style: Style( style.combobox
-                ,   model.isVisible
-                  ? style.visible
-                  : style.hidden
+                , !model.isVisible && style.hidden
```
Comment (Gozala) [inline src/browser/input.js]: This is necessary change.

File: src/browser/input.js [hunk 2]
```diff
@@ -155,32 +155,36 @@ const readSelect = compose
...
     MozAppearance: 'none',
-    height: inputHeight,
-    lineHeight: inputHeight,
+    height: `${inputHeight - 6}px`,
+    lineHeight: `${inputHeight - 6}px`,
     margin: 0,
-    padding: 0,
-    width: `calc(${inputWidth} - ${32 * 2}px)`
+    padding: `0 ${inputXPadding}px`,
+    width: `calc(${inputWidth - 6}px - ${inputXPadding * 2}px)`
```
Comment (Gozala) [inline src/browser/input.js]: Seem to me that we don't need calc as we can calculate that directly.

File: src/browser/input.js [hunk 3]
```diff
@@ -155,32 +155,36 @@ const readSelect = compose
...
+const inputXPadding = 32;
 
 const style = StyleSheet.create({
   combobox: {
-    background: '#EBEEF2',
-    borderRadius: '5px',
     height: inputHeight,
     left: '50%',
-    marginLeft: `calc(-1 * (${inputWidth} / 2))`,
+    marginLeft: `calc(-1 * (${inputWidth}px / 2))`,
```
Comment (Gozala) [inline src/browser/input.js]: Seems like we don't need a calc here either as we can calculate it directly.

File: src/browser/input.js (hunk 2 above)
Comment (gordonbrander) [inline src/browser/input.js]: Oops, you are right. This is a mistake left over from the previous implementation.

File: src/browser/input.js (hunk 3 above)
Comment (gordonbrander) [inline src/browser/input.js]: Ditto.

File: src/browser/input.js (hunk 1 above)
Comment (gordonbrander) [inline src/browser/input.js]: No, just snuck it in because I was already in the file.

**Pull request merged**